resource_booking/models/calendar_event.py

```diff
@@ -52,6 +52,24 @@ def write(self, vals):
         rescheduled._validate_booking_modifications()
         return result
 
+    def create_attendees(self):
+        """Autoconfirm resource attendees if preselected."""
+        old_attendees = self.attendee_ids
+        result = super().create_attendees()
+        new_attendees = self.attendee_ids - old_attendees
+        for attendee in new_attendees:
+            # No need to change state if it's already done
+            if attendee.state in {"accepted", "declined"}:
```

Comment:
Should we use `filtered` in the loop for avoiding 2 lines of code?

Reply:
The performance reasoning is not true: cache makes the "double" loop to not impact. About the readability, I prefer one line with the criteria that 3. Lastly, with the `filtered` option, you don't have uncovered lines by tests.

Reply:
> Lastly, with the `filtered` option, you don't have uncovered lines by tests

That'd be a false positive.

Reply:
I don't think so, but OK, we will do this differently when coding our PRs.